bgc_derivation/processo2.m

```diff
@@ -15,6 +15,7 @@ function processo2(floatnos,varargin)
 %      Also no wiring yet for certificate values
 
 %  title - s processo2  vr - 1.0  authot - bodc/sgl  date - 20211112
+%                       vr -1.0.1 authot -bodc/vidkri date - 20020816 added the case 201_202_202
```

Comment:
spelling 'authot'

Reply:
removed this line as it will be captured in github